app/components/OurDomainsSection/OurDomainsSection.tsx

```diff
@@ -0,0 +1,150 @@
+import Image from "next/image";
+import { useState } from "react";
+import { ArrowButton, BorderedContainer, Button, H2, H5 } from "components";
+import { cn } from "utils/cn";
+import { useWindowSize } from "hooks";
+
+import web3DomainSrc from "./images/web3Domain.png";
+import corporateDomainSrc from "./images/corporateDomain.png";
+import bitcoinsDomainSrc from "./images/bitcoinsDomain.png";
+import ecommerceDomainSrc from "./images/ecommerceDomain.png";
+
+export const OurDomainsSection = () => {
+  const [activeFilter, setActiveFilter] = useState("all");
+  const { isLaptopS } = useWindowSize();
+  const isAll = activeFilter === "all";
+  const isWeb3 = activeFilter === "web3";
+  const isFinance = activeFilter === "finance";
+  const isEcommerce = activeFilter === "e-commerce";
+
+  const handleClickFilter = (filter: string) => {
+    setActiveFilter(filter);
+  };
+
+  return (
+    <section className="flex flex-col items-center lg:my-[120px] sm:my-[100px] my-[60px]">
+      <div className="flex justify-between w-full items-end sm:items-center gap-4 sm:flex-row flex-col sm:mb-[60px] mb-6">
+        <H2 className="sm:w-fit w-full">Our domains</H2>
+        <div className="flex">
+          <Button
+            onClick={() => handleClickFilter("all")}
+            size="filter"
+            className={cn("rounded-s-2xl", {
+              "text-black": isAll,
+            })}
+          >
+            All
+          </Button>
+          <Button
+            onClick={() => handleClickFilter("web3")}
+            size="filter"
+            className={cn({
+              "text-black": isWeb3,
+            })}
+          >
+            Web3
+          </Button>
+          <Button
+            onClick={() => handleClickFilter("finance")}
+            size="filter"
+            className={cn({
+              "text-black": isFinance,
+            })}
+          >
+            Finance
+          </Button>
+          <Button
+            onClick={() => handleClickFilter("e-commerce")}
+            size="filter"
+            className={cn("rounded-e-2xl", {
+              "text-black": isEcommerce,
+            })}
+          >
+            E-commerce
+          </Button>
+        </div>
+      </div>
+      <div className="flex flex-col gap-9">
+        {isAll || isWeb3 ? (
```

Comment:
Component is not a really separated, a lot of code is repeated, let's please split it and create a `Domain` component which we will use, because most of the things are repeated